### ISSUE TITLE
A11y analysis

### ISSUE THREAD

**Issue by tomByrer** [forem/forem#7139]
From here: https://github.com/a11y-reviews/a11y.reviews/blob/master/index.md#devto

> 31 January 2019... The overall interface contains many links without accessible names, some of them apparently only existing to enlarge the click area - see the article teaser listing. Said teaser listing also infinite scrolling.

> Regarding keyboard accessibility: outline on focus is not disabled via CSS, but some elements are reachable only on hover (example: the "Follow" control inside a tag list item). Said control, again, is an anchor element with a `href="#"` but should be a button.

**Comment by benhalpern** [comment]
Thanks for pointing this out. WCAG compliance is where we want to be and being on this list is a really bad thing. We'll take this as a note to prioritize closing the loop on some of our accessibility cases like described here with the follow button only accessible via hover. Thank you very much for pointing this out.

**Comment by tomByrer** [comment]
Good news; your site has a screenshot with an AAA rating in an article about accessibility at Google's web.dev:

https://web.dev/a11y-tips-for-web-dev/

**Comment by marcysutton** [comment]
I've been looking to see if any of these items are still outstanding and here's what I found:

- Infinite scrolling is definitely still an issue, but it could be tracked separately in https://github.com/forem/forem/issues/5670.
- Regarding links without accessible names, I remember those as well but I believe most have been fixed. I'm seeing two right now on the DEV homepage that are both coming from a sponsorship widget. I've raised that as an issue internally to start, and will file a separate issue for it if needed.
- I believe the hover-only follow control issue has been fixed, but I'm not 100% sure I was looking at the right thing. @nickytonline or @benhalpern do you know what they meant by "the "Follow" control inside a tag list item"?

I did [create a PR](https://github.com/forem/forem/pull/11721) to address some semantics things I found, like heading order and some missing alt text / form labels. But I think we're near being able to close this issue.

**Comment by cmgorton** [comment]
Thank you @tomByrer for opening this issue. Since we have Marcy working on identifying a11y issues and creating other issues/PRs to monitor those I am going to go ahead and close this issue.
